recipes-kernel/linux/linux-raspberrypi.inc

```diff
@@ -27,7 +27,8 @@ LINUX_VERSION_EXTENSION ?= ""
 
 # CMDLINE for raspberrypi
 SERIAL = "${@oe.utils.conditional("ENABLE_UART", "1", "console=serial0,115200", "", d)}"
-CMDLINE ?= "dwc_otg.lpm_enable=0 ${SERIAL} root=/dev/mmcblk0p2 rootfstype=ext4 rootwait"
+CMA = "${@oe.utils.conditional("RASPBERRYPI_CAMERA_V2", "1", "cma=64M", "", d)}"
```

Comment:
Why do we need this?

Reply:
For camera device, minimum of 64M contiguous memory allocation is needed.
So, we have added this in the cmdline.txt.

Reply:
We had tried keeping the cma as 32M, and libcamera could not able to capture the HD resolution 1280x720.

We are getting the following error.
ERROR V4L2 v4l2_videodevice.cpp:1129 /dev/video0[cap]: Not enough buffers provided by V4L2VideoDevice
ERROR RPI raspberrypi.cpp:781 Failed to allocate buffers

When we try to capture in HD resolution 1280x720, libcamera is selecting the best resolution as 1920x1080.
And since libcamera is capturing the frames in 10 bit bayer format following memory is required by it.

1920x1080x10/8 = 2592000 bytes.

So libcamera is requesting for 4 buffers with above mentioned size.

So we considered, minimum of 64M for CMA is required to work with libcamera for HD streams.

Reply:
For reference, I believe when the firmware/GPU based camera drivers are used, they reserve 128MB of RAM (gpu_mem=128m)

Reply:
Thanks @kbingham 
@agherzan  this is a new knob and is only activated when using `RASPBERRYPI_CAMERA_V2` I think we should be fine with this change

Reply:
It's not really a new knob - it's a definition based on another "knob" :) But overall I'm fine with this. I was thinking if we could make it a bit more elegant. Let the builds crunch it and I'll merge afterwards.